### DISCUSSION TITLE
DataGrid for Uno v5.3 - what package can be used that support it...

### DISCUSSION THREAD

**Discussion by datovy-edam** [unoplatform/uno#18985] [Q&A] [answered]
### What do you need help with?

I am trying to identify in what CommunityToolkit the DataGrid is available and usable for Uno v5.3 and/or .net8?

Which package should be used?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! You can find the steps to install community toolkit datagrid with [the 7.x steps here](https://platform.uno/docs/articles/uno-community-toolkit-v7.html?tabs=tabid-winui%2Csingleproj). Are you having issues with later versions of Uno, if you're staying on 5.3?

**Comment by datovy-edam** [comment]
Thanks for the quick response!

While following the given instructions... so far all looks good.  Thanks.
